test/TEST-15-DROPIN/test-dropin.sh

```diff
@@ -0,0 +1,277 @@
+#! /bin/bash
+
+set -e
+
```

Comment:
I think we should always add `set -x`. This helps to debug failures.
`test-dropin.sh` fails with:
```sh
...
Feb 01 16:56:28 test-dropin.sh[27]: + mkdir -p /usr/lib/systemd/system/a.service.wants /usr/lib/systemd/system/a.service.requires
Feb 01 16:56:28 test-dropin.sh[27]: + ln -s ../unexisting.service /etc/systemd/system/a.service.wants/
Feb 01 16:56:28 test-dropin.sh[27]: + check_ok a Wants unexisting.service
Feb 01 16:56:28 test-dropin.sh[27]: + '[' 3 -eq 3 ']'
Feb 01 16:56:28 test-dropin.sh[27]: + case $(systemctl show -p$2 $1) in
Feb 01 16:56:28 test-dropin.sh[27]: ++ systemctl show -pWants a
Feb 01 16:56:28 test-dropin.sh[27]: + return 0
Feb 01 16:56:28 test-dropin.sh[27]: + systemctl start a
Feb 01 16:56:28 test-dropin.sh[27]: + systemctl stop a
Feb 01 16:56:28 test-dropin.sh[27]: + echo '  test 4'
Feb 01 16:56:28 test-dropin.sh[27]:   test 4
Feb 01 16:56:28 test-dropin.sh[27]: + ln -sf ../unexisting.service /etc/systemd/system/a.service.requires/
Feb 01 16:56:28 test-dropin.sh[27]: + check_ok a Requires unexisting.service
Feb 01 16:56:28 test-dropin.sh[27]: + '[' 3 -eq 3 ']'
Feb 01 16:56:28 test-dropin.sh[27]: + case $(systemctl show -p$2 $1) in
Feb 01 16:56:28 test-dropin.sh[27]: ++ systemctl show -pRequires a
Feb 01 16:56:28 test-dropin.sh[27]: + return 1
```